modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/config/StorageTarget.scala

```diff
@@ -88,6 +98,36 @@ object StorageTarget {
     }
   }
 
+  final case class Databricks(
+                               host: String,
+                               schema: String,
+                               port: Int,
+                               httpPath: String,
+                               password: PasswordConfig,
+                               sshTunnel: Option[TunnelConfig]
+                             ) extends StorageTarget {
+
+    override def username: String = "token"
+
+    override def driver: String = "com.simba.spark.jdbc.Driver"
+
+    override def connectionUrl: String = s"jdbc:spark://$host:$port/$schema"
+
+    override def doobieCommitStrategy: Strategy   = Strategy.void
+    override def doobieNoCommitStrategy: Strategy = Strategy.void
+    override def withAutoCommit                   = true
+
+    override def properties: Properties = {
+      val props: Properties = new Properties()
+      props.put("httpPath", httpPath)
+      props.put("ssl", 1)
+      //      props.put("LogLevel", 6)
+      props.put("AuthMech", 3)
+      props.put("transportMode", "http")
```

Comment:
We will need to set a property "UserAgentEntry" to identify ourselves to Databricks, much like we do [here for Snowflake](https://github.com/snowplow/snowplow-rdb-loader/blob/3.0.1/modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/config/StorageTarget.scala#L114).  I will find out what the default needs to be.